commands/add.go

```diff
@@ -15,7 +15,7 @@ const (
 )
 
 func init() {
-	RootCmd.AddCommand(&cobra.Command{
+	rootCmd.AddCommand(&cobra.Command{
```

Comment:
No worries.

I'll let this one stand, but I'll make a better attempt in the future to keep these PRs better scoped.